packages/shared/src/components/modals/referral/SearchReferralModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Modal, ModalProps } from '../common/Modal';
+import { Checkbox } from '../../fields/Checkbox';
+import { Button } from '../../buttons/Button';
+import CopyIcon from '../../icons/Copy';
+import { useSettingsContext } from '../../../contexts/SettingsContext';
+import { CampaignCtaPlacement } from '../../../graphql/settings';
+import useMedia from '../../../hooks/useMedia';
+import { laptop } from '../../../styles/media';
+import { cloudinary } from '../../../lib/image';
+import { KeysRow } from './KeysRow';
+import { ReferralCampaignKey, useReferralCampaign } from '../../../hooks';
+
+function SearchReferralModal(modalProps: ModalProps): ReactElement {
+  const isLaptop = useMedia([laptop.replace('@media ', '')], [true], false);
+  const { campaignCtaPlacement, onToggleHeaderPlacement } =
+    useSettingsContext();
+  const { availableCount, noKeysAvailable } = useReferralCampaign({
+    campaignKey: ReferralCampaignKey.Search,
+  });
+
+  return (
+    <Modal
+      {...modalProps}
+      kind={isLaptop ? Modal.Kind.FixedCenter : Modal.Kind.FlexibleCenter}
+      size={isLaptop ? Modal.Size.XLarge : Modal.Size.Small}
+    >
+      <Modal.Body className="laptop:flex-row">
+        <span className="laptop:hidden -mx-6 -mt-6">
+          <img
+            src={cloudinary.referralCampaign.search.bgPopupMobile}
+            alt="tablet popup background"
+          />
+        </span>
+        <div className="flex flex-col laptop:p-2 w-full max-w-[26rem]">
+          <h1 className="laptop:mt-4 font-bold text-center laptop:text-left typo-title3 tablet:typo-title1 laptop:typo-mega2">
+            {noKeysAvailable
+              ? `Need more keys?`
+              : `Give your friends early access to daily.dev's search!`}
+          </h1>
+          <p className="mt-6 text-center laptop:text-left !font-normal typo-body tablet:typo-headline laptop:typo-title3 text-theme-label-secondary">
```

Comment:
Why do you need the override on normal, which one adds bold by default?

Reply:
The `typo-headline` automatically applies bold.